src/main/java/org/gridsuite/study/server/NetworkStoreService.java

```diff
@@ -97,4 +98,40 @@ private String getNetworkStoreServerServerURI() {
                 .map(e -> VoltageLevelInfos.builder().id(e.getId()).name(e.getAttributes().getName()).substationId(e.getAttributes().getSubstationId()).build())
                 .collect(Collectors.toList()));
     }
+
+    // This function call directly the network store server without using the dedicated client because it's a blocking client.
+    // If we'll have new needs to call the network store server, then we'll migrate the network store client to be nonblocking
+    Mono<List<BusInfos>> getVoltageLevelBuses(UUID networkUuid, String voltageLevelId, int variantNum) {
+        String path = UriComponentsBuilder.fromPath("{networkId}/{variantNum}/voltage-levels/{voltageLevelId}/configured-buses")
```

Comment:
{networkId} -> {networkUuid} ? or {1}, {2}, {3} because its seems to be managed by order in buildAndExpand ?

Reply:
same for deleteNetwork and getNetworkVoltageLevels nope ?